tests/integration_tests/test_class_imbalance_feature.py

```diff
@@ -0,0 +1,189 @@
+import contextlib
+import os
+import shutil
+import tempfile
+
+import numpy as np
+import pandas as pd
+import pytest
+import ray
+
+from ludwig.api import LudwigModel
+from ludwig.backend import LocalBackend
+from ludwig.backend.ray import RayBackend
+from tests.integration_tests.utils import create_data_set_to_use, spawn
+
+RAY_BACKEND_CONFIG = {
```

Comment:
My sense is that we can probably remove ray backends from all of these tests since class imbalance should work independent of which backend we're using, and there's also no ray-specific configuration parameters.

Ray tests are slower and the additional overhead for setting up makes these tests harder to read.

Using local backends here will keep the tests simple, and if we wanted to test the ray+imbalance combination, we should do so in `test_ray.py`.

Reply:
Great idea, consolidating all the tests now!